aspnetcore/blazor/fundamentals/additional-scenarios.md

```diff
@@ -218,6 +218,36 @@ Blazor.defaultReconnectionHandler._reconnectionDisplay =
 
 The placeholder `{ELEMENT ID}` is the ID of the HTML element to display.
 
+::: moniker range=">= aspnetcore-5.0"
+
+## Influence HTML `<head>` tag elements
+
+*This section applies to Blazor WebAssembly and Blazor Server.*
+
+When rendered, the `Title`, `Link`, and `Meta` components add or update data in the HTML `<head>` tag elements:
+
+```razor
+@using Microsoft.AspNetCore.Components.Web.Extensions.Head
+
+<Title Value="{TITLE}" />
+<Link href="{URL}" rel="stylesheet" />
+<Meta content="{DESCRIPTION}" name="description" />
+```
+
+In the preceding example, placeholders for `{TITLE}`, `{URL}`, and `{DESCRIPTION}` are sting values, Razor variables, or Razor expressions.
```

Comment:
`sting` -> `string` on line 237

(not sure I can directly edit this)

Reply:
I'll fix it. You can't edit directly ... it's a bit of a pain, but you have to branch from the PR's branch and then submit a PR back to it. It's faster 🏃 if I just edit directly. Thanks for calling it out.